.claude/skills/presentation-design/scripts/style_matcher.py

```
@@ -0,0 +1,263 @@
+"""
+    
+    return guide
+
+
+if __name__ == "__main__":
+    # Test the matcher
+    print("=== Style Matcher Test ===\n")
+    
+    # Test case 1: AI topic
+    result = match_style("AI 윤리", "학회발표")
+    print(f"Topic: AI 윤리")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)